### PR TITLE
Replace some references to sf.net

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: contrib/index_git.html [hunk 1]
```diff
@@ -497,22 +497,19 @@ <h3>Downloading a copy</h3>
 
 If you really need the latest version get it from git:
 <pre>
-git clone git://notion.git.sourceforge.net/gitroot/notion/contrib
+git clone git@github.com:raboof/notion
```
Comment (knixeur) [inline contrib/index_git.html]: Same as above use ```git clone https://github.com/raboof/notion.git``` instead

File: contrib/index_git.html [hunk 2]
```diff
@@ -497,22 +497,19 @@ <h3>Downloading a copy</h3>
...
 </pre>
 This will get you a fully functional copy of the repository in the
-directory <tt>contrib</tt>. 
+directory <tt>notion/contrib</tt>.
```
Comment (knixeur) [inline contrib/index_git.html]: I think this should stay as contrib/ as it's the directory inside the repository (notion)

File: README.md [hunk 3]
```diff
@@ -3,21 +3,17 @@ Notion
...
 Building and installing
 -----------------------
 
-See also: 
-
-  https://sourceforge.net/apps/mediawiki/notion/index.php?title=Development
-
 1. Get the source code.
-    git clone git://notion.git.sourceforge.net/gitroot/notion/notion
+    git clone git@github.com:raboof/notion
```
[the remark this answers is not in the document]
Comment (raboof) [reply README.md]: Good points, updated

File: contrib/index_git.html (hunk 2 above)
Comment (raboof) [reply contrib/index_git.html]: This is relative to the directory where you did the 'git clone', so I think 'notion/contrib' is accurate here.

**Comment by knixeur** [comment]
LGTM

**Pull request merged**